### PR TITLE
Adding command injection entry

### PR DESCRIPTION
**Pull request by PiranhaSa**

### CODE REVIEW

**Review by benyissa — COMMENTED** [review]
in the recommendation part. Shouldn't we add extra examples like C code?

File: WEB_SERVICE/WEB/_HIGH/COMMAND_INJECTION/meta.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "risk_rating": "high",
```
Comment (3asm) [inline WEB_SERVICE/WEB/_HIGH/COMMAND_INJECTION/meta.json:2]: critical

**Review by 3asm — APPROVED** [review]

**Comment by PiranhaSa** [comment]
> in the recommendation part. Shouldn't we add extra examples like C code?

Not really , because we are talking about the web .

File: WEB_SERVICE/WEB/_HIGH/COMMAND_INJECTION/meta.json (hunk 1 above)
Comment (PiranhaSa) [reply WEB_SERVICE/WEB/_HIGH/COMMAND_INJECTION/meta.json]: done

**Review by BlueSquare1 — APPROVED** [review]

**Review by elyousfi5 — APPROVED** [review]
lgtm, please make sure to pull the changes in OSS and RE for the KB submodule

**Pull request merged**